### PR TITLE
fix: metrics test flakiness

### PR DESCRIPTION
**Pull request by philipyoo**

### CODE REVIEW

File: tests/functional/api/BackbeatServer.js [hunk 1]
```diff
@@ -178,42 +184,33 @@ describe('Backbeat Server', () => {
...
         done => {
-            statsClient.reportNewRequest(OPS, 5);
-            statsClient.reportNewRequest(BYTES, 7100);
-
             getRequest('/_/metrics/backlog', (err, res) => {
                 assert.ifError(err);
                 const key = Object.keys(res)[0];
-                assert.equal(res[key].results.count, 5);
-                assert.equal(res[key].results.size, 7.10);
+                assert.equal(res[key].results.count, 1275);
```
Comment (JianqinWang) [inline tests/functional/api/BackbeatServer.js]: in line 138: ```statsClient.reportNewRequest(OPS, 1725);```
should these 2 numbers be the same?

Comment (philipyoo) [reply tests/functional/api/BackbeatServer.js]: Thanks for checking, in this case, I went ahead and selected fixed numbers for all tests to use, so the change here is accurate 👍

File: tests/config.json [hunk 2]
```diff
@@ -31,15 +31,21 @@
         "replication": {
             "topic": "backbeat-test-replication",
             "replicationStatusTopic": "backbeat-test-replication-status",
-            "groupId": "backbeat-test-replication-group"
+            "groupId": "backbeat-test-replication-group",
+            "destination": {
```
Comment (jonathan-gramain) [inline tests/config.json]: Maybe you didn't intend to commit this? Though in practice should not be an issue.

Comment (philipyoo) [reply tests/config.json]: ah yes, I think I was getting ahead of myself here, these changes are already reflected for PR#193 and I think I included this here without much thought, sorry about that

**Comment by jonathan-gramain** [comment]
@ironman-machine r+

**Comment by jonathan-gramain** [comment]
CI tests succeeded in http://ci.ironmann.io/gh/scality/Integration/20314

**Comment by jonathan-gramain** [comment]
@ironman-machine r-

**Pull request merged**